packages/lodestar/src/chain/blocks/validate.ts

```diff
@@ -25,7 +25,7 @@ export function validateBlock(
         if (
           finalizedCheckpoint &&
           finalizedCheckpoint.epoch > 0 &&
-          computeEpochAtSlot(config, job.signedBlock.message.slot) <= finalizedCheckpoint.epoch
+          computeEpochAtSlot(config, job.signedBlock.message.slot) < finalizedCheckpoint.epoch
```

Comment:
this is the same concern I have below. When I start from finalized state of slot 671 (which is epoch 20), the checkpoint is like:
```
{root: hashTreeRoot(block 671), epoch: 21}
```
next slots are 675+ which is epoch 21

Reply:
initially I don't have this change :).

I can't sync because initial sync doesn't received "processedBlock" event if I don't change that. The reason is like I explained above:
+ we start with slot 671, *but* finalized epoch is 21 (due to the concern I have), not 20 (which is the real epoch of slot 671)
+ initial sync fetches blocks 675+ which is epoch 21